### PR TITLE
Avoid empty iso3 codes from synonyms

### PR DESCRIPTION
**Pull request by iriberri**
@rafatower this should clean the emptyness 

    select distinct(rank) from admin0_synonyms where adm0_a3 is null
    > 2

### CODE REVIEW

**Comment by iriberri** [comment]
BTW, I only added ISO3 codes in the source file for those that make sense in our system.

File: geocoder/admin0/data/wikipedia_countries_native_names.csv [hunk 1]
```diff
@@ -46,15 +46,15 @@ TLS,East Timor,Timor Lorosa'e
 BHS,Bahamas,The Bahamas
 CHN,China,Zhōngguó
 EGY,Egypt,Misr or Masr
-,Equatorial Guinea,Guinea Ecuatorial
+GNQ,Equatorial Guinea,Guinea Ecuatorial
```
Comment (rafatower) [inline geocoder/admin0/data/wikipedia_countries_native_names.csv]: modifying this file looks like "cheating to me" but ok if you know what you're doing :)

I mean, this is a raw source file, right?

File: geocoder/admin0/sql/build_synonym_table.sql [hunk 2]
```diff
@@ -37,7 +37,9 @@ INSERT INTO admin0_synonyms (name, rank, adm0_a3)
 SELECT 
     country_endonym, 2,  adm0_a3
 FROM
-    wikipedia_countries_native_names;
+    wikipedia_countries_native_names
+WHERE
+    adm0_a3 IS NOT null;
```
Comment (rafatower) [inline geocoder/admin0/sql/build_synonym_table.sql]: this is the important change :+1:

File: geocoder/admin0/data/wikipedia_countries_native_names.csv (hunk 1 above)
Comment (iriberri) [inline geocoder/admin0/data/wikipedia_countries_native_names.csv]: This was not got from a raw source, just copy-pasting things from the Wikipedia. I only added official ISO3 codes, I avoided other ones that we're using but are not that official.

Comment (rafatower) [inline geocoder/admin0/data/wikipedia_countries_native_names.csv]: ok, thanks for the clarification.

**Comment by rafatower** [comment]
:+1: looks good to me

**Pull request merged**